src/site/content/en/en.11tydata.js

```diff
@@ -18,6 +18,7 @@ module.exports = function () {
     allPaths['fast'],
     allPaths['reliable'],
     allPaths['secure'],
+    allPaths['mini-apps'],
```

Comment:
I think you can remove this.

This file controls the first 6 cards that we display on the homepage

Reply:
Done!